C/CryptoMiniSat/build_tarballs.jl

```diff
@@ -31,10 +34,20 @@ make install
 install_license ${WORKSPACE}/srcdir/cryptominisat/LICENSE.txt
 """
 
+augment_platform_block = """
+    using Base.BinaryPlatforms
+    $(MPI.augment)
+    augment_platform!(platform::Platform) = augment_mpi!(platform)
+"""
+
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = expand_cxxstring_abis(supported_platforms())
 
+# We need to expand the gfortran versions because MPItrampoline
+# depends on Fortran.
+platforms = expand_gfortran_versions(platforms)
```

Comment:
You don't need to do this if the library itself doesn't link to libgfortran, unless I'm missing something

Reply:
I think there was a problem that went away when I added this.

I'll remove it. If necessary, I will add it back in with a better explanation...

Reply:
Removed.